src/javascript/core/send.js

```diff
@@ -75,15 +77,27 @@ function sendRequest(request, callback) {
 		}
 
 		if (error) {
-			// Re-add to the queue if it failed.
-			// Re-apply queueTime here
-			request.queueTime = queueTime;
-			queue.add(request).save();
-
-			utils.broadcast('oErrors', 'log', {
-				error: error.message,
-				info: { module: 'o-tracking' }
-			});
+			// If IE11 XHR error, try using image method
+			if (ie11() && transport.name === 'xhr') {
+				let image_method = transports.get('image')();
+				request.system.transport = [request.system.transport,image_method.name].join('-'); // Append image label to transport value
```

Comment:
I would put this comment above the line for legibility. Also... It doesn't explain why the image label is assigned

Reply:
Updated to be clearer